onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -1509,11 +1511,28 @@ def aten_embedding_sparse_backward(
     raise NotImplementedError()
 
 
-def aten_empty_like(self: TensorType, memory_format: Optional[str] = None) -> TensorType:
+"""
+@torch_op("aten::empty")
+def aten_empty(size: INT64, dtype : int = 1) -> TensorType:
+    # empty(SymInt[] size, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None, MemoryFormat? memory_format=None) -> Tensor
+
+    # using RandomUniform value to simulate np.empty()
+
+    result = op.RandomUniform(dtype, high=1e30, low=-1e30, shape=size)
```

Comment:
size is an integer value, but the attribute of shape requires a list. So, we could not use it directly.

Reply:
use Zeros_like instead.